cram_core/cram_occasions_events/src/tasks.lisp

```diff
@@ -44,173 +44,191 @@
 (defun task-children (task)
   (mapcar #'cdr (task-tree-node-children task)))
 
-;; Note: #demmeln: There is much potential for optimization, e.g. chaching the
```

Comment:
I would like to keep the original implementation from Niko and Lorenz in this file at the bottom commented out.